.travis.yml

```diff
@@ -77,18 +77,20 @@ after_success:
     if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       # macpython 3.5 doesn't support recent TLS protocols which causes twine
       # upload to fail, so we use the system Python to run twine
-      virtualenv -p /usr/bin/python .twine-env && . .twine-env/bin/activate
+      /usr/bin/python3 -m venv .twine-env && . .twine-env/bin/activate
```

Comment:
Sorry, Apple only ships /usr/bin/python as a symlink to /usr/bin/python2.7.

Reply:
Can the Travis documentation be any vaguer?  https://docs.travis-ci.com/user/reference/osx/#python-related-tools

Reply:
Agreed, it's pretty sparse.

I looked into using the homebrew addon to install Python, but it seems that homebrew [doesn't give control over the Python version](https://docs.brew.sh/Homebrew-and-Python) --- you get 2 or 3, and that's as fine as it goes.

However, if homebrew can install pyenv, then pyenv can install a specific Python version (I use pyenv to install specific Python versions for gevent).  One nice thing is that pyenv takes care to build Python using a functioning OpenSSL on macOS. I just don't know much about homebrew or the specific incantations that would be required here.